packages/manager/apps/carrier-sip/src/index.js

```diff
@@ -11,18 +11,20 @@ import angular from 'angular';
 // Module dependencies.
 import ovhManagerCarrierSip from '@ovh-ux/manager-carrier-sip';
 import uiRouter from '@uirouter/angularjs';
+import { Environment } from '@ovh-ux/manager-config';
 
 import cdr from './cdr';
 import endpoints from './endpoints';
 
 // Routing and configuration.
 import routing from './routing';
-import { momentConfiguration } from './config';
 
 // Styles.
 import '@ovh-ux/ui-kit/dist/css/oui.css';
 
 angular
   .module('carrierSipApp', [cdr, endpoints, ovhManagerCarrierSip, uiRouter])
   .config(routing)
-  .config(momentConfiguration);
+  .config(() => {
+    moment.locale(Environment.getUserLanguage());
```

Comment:
Wherever `moment` is configured via `angular.config`, you're still using `Environment` and not `coreConfig`.
Is it a normal behavior?

Reply:
Yes, it's a _normal behavior_ for me, because in the second PR (https://github.com/ovh/manager/pull/4505), I updated the app bootstrap to inject the environment and I prefer to use the env directly instead of using the `coreConfigProvider` in this specific case.
I will also update the moment locale init to load locale on-demand https://github.com/ovh/manager/blob/feat%2Fufrontend-env-apps/packages/manager/apps/carrier-sip/src/app.module.js#L36-L50